lib/src/io/vm.dart

```diff
@@ -82,8 +82,7 @@ DateTime modificationTime(String path) {
   return stat.modified;
 }
 
-String getEnvironmentVariable(String name) =>
-    io.Platform.environment['SASS_PATH'];
```

Comment:
Nothing was relying on this always being 'SASS_PATH'?

Reply:
No, we were always passing it in explicitly as well (which is why it worked in the Node implementation).